.github/workflows/main.yml

```diff
@@ -0,0 +1,66 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+  pull_request:
```

Comment:
Woudn't it be possible to run the ci also on pushes on develop by specifying `master` and `develop` here ? (https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#filtering-for-specific-branches-tags-and-paths)

Reply:
As far as I know, this *will* run the ci on pushes to master and develop (once this is merged).  It should work on any branch.  If we add a filter then I think it *only* runs on those branches.
Let me know if you think I've missed something.

Reply:
Ah right. Thanks